### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by chill-was-taken**
The application of IntelliJ suggestions.

### CODE REVIEW

File: src/main/java/com/glodblock/github/common/parts/PartFluidLevelEmitter.java [hunk 1]
```diff
@@ -132,8 +132,7 @@ private boolean isLevelEmitterOn() {
 
         final boolean flipState = this.getConfigManager().getSetting(Settings.REDSTONE_EMITTER)
                 == RedstoneMode.LOW_SIGNAL;
-        return flipState ? this.reportingValue >= this.lastReportedValue + 1
-                : this.reportingValue < this.lastReportedValue + 1;
+        return flipState == (this.reportingValue >= this.lastReportedValue + 1);
```
Comment (wlhlm) [inline src/main/java/com/glodblock/github/common/parts/PartFluidLevelEmitter.java:135]: I'll be honest, these two lines are too clever for me, We should make this more verbose, extra SLOC be damned, something like:
```java
final boolean flipState = this.getConfigManager().getSetting(Settings.REDSTONE_EMITTER);
if (flipState == RedstoneMode.LOW_SIGNAL) {
    return this.reportingValue >= (this.lastReportedValue + 1);
} else {
    return this.reportingValue < (this.lastReportedValue + 1);
}
```
(please check the condition is correct, I haven't tested this)

File: src/main/java/com/glodblock/github/crossmod/waila/PartWailaDataProvider.java [hunk 2]
```diff
@@ -49,7 +49,7 @@ public ItemStack getWailaStack(final IWailaDataAccessor accessor, final IWailaCo
             ItemStack wailaStack = null;
```
Comment (wlhlm) [inline src/main/java/com/glodblock/github/crossmod/waila/PartWailaDataProvider.java:49]: Similar to [here](https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/133/commits/b77116b584f1682085293a913f2477f2ed9440c9), you can drop `null` assignment.

**Review by wlhlm — CHANGES_REQUESTED** [review]
Nice! Thanks for your continued janitorial work on our codebases. 👍

File: src/main/java/com/glodblock/github/common/parts/PartFluidLevelEmitter.java (hunk 1 above)
Comment (chill-was-taken) [reply src/main/java/com/glodblock/github/common/parts/PartFluidLevelEmitter.java]: Thank you for bringing attention to this. You're right. I fixed it in https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/133/commits/71048a456d9e41f2ac814c9672006226959f3a41

File: src/main/java/com/glodblock/github/crossmod/waila/PartWailaDataProvider.java (hunk 2 above)
Comment (chill-was-taken) [reply src/main/java/com/glodblock/github/crossmod/waila/PartWailaDataProvider.java:49]: Seems like it's not possible.
image

Comment (chill-was-taken) [reply src/main/java/com/glodblock/github/crossmod/waila/PartWailaDataProvider.java:49]: Rephrasing the [following](https://stackoverflow.com/questions/2448843/variable-might-not-have-been-initialized-error) issue,
> Their default value is given when declared as class members. Local variables don't have default values

Comment (wlhlm) [reply src/main/java/com/glodblock/github/crossmod/waila/PartWailaDataProvider.java:49]: Oh, yeah that makes sense.

**Review by wlhlm — APPROVED** [review]

**Pull request merged**